src/factories/interfaces/IImmutable_PIM_Factory_v1.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// Internal Interfaces
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {IOrchestratorFactory_v1} from
+    "src/factories/interfaces/IOrchestratorFactory_v1.sol";
+import {IBondingCurveBase_v1} from
+    "@fm/bondingCurve/interfaces/IBondingCurveBase_v1.sol";
+
+// Internal Dependencies
+import {ERC20Issuance_v1} from "src/external/token/ERC20Issuance_v1.sol";
+
+/**
+ * @title   Immutable PIM Factory
+ *
+ * @notice  Used to deploy a PIM workflow with an unrestricted bonding curve and a mechanism to
+ *          ensure immutability of the workflow while still enabling the claiming of fees.
+ *
+ * @dev     More user-friendly way to deploy a PIM workflow with an unrestricted bonding curve.
+ *          Acts as wrapper for workflow to make it "unruggable" while exposing fee claiming functionality.
+ *
+ * @custom:security-contact security@inverter.network
+ *                          This contract is experimental in nature and has not been audited.
+ *                          Please use at your own risk!
+ *
+ * @author  Inverter Network
+ */
+interface IImmutable_PIM_Factory_v1 {
+    //--------------------------------------------------------------------------
+    // Errors
+
+    /// @notice Error thrown when an unpermissioned address tries to claim fees or to transfer role.
+    error PIM_WorkflowFactory__OnlyPimFeeRecipient();
+
+    /// @notice Error thrown when a zero address is passed as initiator.
+    error PIM_WorkflowFactory__InvalidZeroAddress();
+
+    //--------------------------------------------------------------------------
+    // Events
+
+    /// @notice Event emitted when a new PIM workflow is created.
+    /// @param fundingManager The address of the funding manager.
+    /// @param issuanceToken The address of the issuance token.
+    /// @param deployer The address of the deployer.
+    event PIMWorkflowCreated(
+        address indexed fundingManager,
```

Comment:
I guess this is for indexing reasons, but why reuturn the fundingManager instead of the Orchestrator?

Reply:
The implementation returns the orchestrator address, so thi is a bug :)

Reply:
woopsies